modules/platforms/python/pygridgain/client.py

```diff
@@ -80,10 +80,10 @@ class Client:
     _compact_footer: bool = None
     _connection_args: Dict = None
     _current_node: int = None
-    _nodes: Union[Iterable[Connection], Dict['UUID', Connection]] = None
+    _nodes: List[Connection] = None
```

Comment:
In case of looking up the connection by its UUID − yes, `dict` lookup would be faster. But I came to conclusion that I need a uniform storage for connections that works whether they have UUIDs or not (thin protocol 1.3- or 1.4+). Then iterating `list` should be faster then determining the proper procedure for addressing the storage depending on protocol version.